src/features/Instructor/AddEvent/index.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Select, Button } from 'react-paragon-topaz';
+
+import {
+  ModalDialog, ModalCloseButton, Form, Col,
+} from '@edx/paragon';
+
+const initialStateModal = {
+  type: 'not-available',
+  allDay: true,
+  repeat: { label: 'Does not repeat', value: 'not-repeat' },
+  startDate: '',
+  endDate: '',
+  startHour: '',
+  endHour: '',
+};
+
+const typeEventOptions = [
+  { label: 'Not available', value: 'not-available' },
+  { label: 'Available', value: 'available' },
+  { label: 'Prep Time', value: 'prep-time' },
+];
+
+const repeatOptions = [
+  { label: 'Does not repeat', value: 'not-repeat' },
+  { label: 'Repeat weekly', value: 'weekly' },
+  { label: 'Repeat monthly', value: 'monthly' },
+];
+
+const AddEvent = ({ isOpen, onClose }) => {
+  const [eventData, setEventData] = useState({ ...initialStateModal });
+
+  const customStyles = {
+    control: (baseStyles, state) => ({
+      ...baseStyles,
+      border: 0,
+      boxShadow: state.isFocused ? 0 : 0,
+    }),
+    indicatorsContainer: (baseStyles) => ({
+      ...baseStyles,
+      alignItems: 'baseline',
+    }),
+    singleValue: (baseStyles) => ({
+      ...baseStyles,
+      color: '#666',
+      fontSize: '1rem',
+      fontWeight: '700',
+    }),
+  };
+
+  useEffect(() => {
+    if (!isOpen) {
+      setEventData(initialStateModal);
+    }
+  }, [isOpen]);
+
+  return (
+    <ModalDialog
+      title="New event"
+      isOpen={isOpen}
+      onClose={onClose}
+      hasCloseButton
+    >
+      <ModalDialog.Header>
+        <ModalDialog.Title>New Event</ModalDialog.Title>
+      </ModalDialog.Header>
+      <ModalDialog.Body>
+        <Form>
+          <Form.Row className="col-12 px-0">
+            <Form.Group>
+              <Form.RadioSet
+                name="event"
+                className="d-flex align-items-end justify-content-between col-12 px-1"
+                onChange={e => setEventData({ ...eventData, type: e.target.value })}
+                isInline
+                value={eventData.type}
+              >
+                {typeEventOptions.map(option => (
+                  <Form.Radio value={option.value} key={option.value}>{option.label}</Form.Radio>
+                ))}
+              </Form.RadioSet>
+            </Form.Group>
+          </Form.Row>
+          <Form.Row>
+            <Form.Group className="d-flex align-items-center justify-content-between col-12">
+              <Form.Switch
```

Comment:
I got a few concerns about this:
![image](https://github.com/user-attachments/assets/48d4ed5d-a760-4e25-8e2e-7e32a67f8bbf)

1. Is not possible to add an icon if is active or inactive?
2. If is active, is possible to remove that green color, definitely does not match with the design :face_with_diagonal_mouth:

Reply:
I change the green color, the icon is not possible